src/bootstrap/dist.rs

```diff
@@ -518,8 +516,7 @@ impl Step for Rustc {
                     .join("bin")
                     .join(&exe);
                 // for the rationale about this rename check `compile::copy_lld_to_sysroot`
```

Comment:
I suppose this part only should be reverted, could you check it on windows?

Reply:
Unfortunately I don't have a Windows machine to test on, so I'm just trying to fix the regession quickly.